app/views/Project/Tagging/index.tsx

```diff
@@ -130,6 +130,10 @@ function Tagging(props: Props) {
                     >
                         {subNavbarComponents}
                     </Route>
+                    <Route
+                        exact
+                        path={routes.assessmentEdit.path}
+                    />
```

Comment:
This route is here to load the subNavbarComponents. Please pass them as children